README.md

```diff
@@ -29,6 +29,10 @@ supported).
    would add significant overhead when calling into and out of extension
    modules.
 
+3. C locale support: Go doesn't support locales in the same way that C does. As such,
```

Comment:
I think this is more like something that isn't *yet* supported. I suspect we'll need to support some variation on locales.